CMakeSettings.json

```diff
@@ -3,10 +3,10 @@
     {
       "name": "Release",
       "configurationType": "Release",
-      "generator": "Visual Studio 16 2019 Win64",
+      "generator": "Ninja",
```

Comment:
I'd be surprised if the previous Visual Studio specifier even worked. Since VS 2019, in CMake, generation needs to be specified like:

```cmake
cmake -G "Visual Studio 16 2019" -A x64
```

`Win64` isn't a thing anymore for the generator